rsrc/dialogs/restart-game.xml

```diff
@@ -3,8 +3,8 @@
 <?xml-stylesheet href="dialog.xsl" type="text/xsl"?>
 <dialog defbtn='cancel'>
 	<pict type='dlog' num='23' top='9' left='9'/>
-	<text top='6' left='53' width='249' height='31'>
-		Starting over will delete the current party.
+	<text name='warning' top='6' left='53' width='249' height='31'>
+		{{action}} will unload the current party without saving.
```

Comment:
I actually don't like this phrasing. I can sort of understand what was bad about the old phrasing, but this might be even worse in different ways. My main gripe is that "unload" feels a bit too technical. I think it should be phrased more along the lines of "You will lose unsaved changes if". Maybe not _exactly_ like that but closer to that.

Side note: This is also unlocalizable. That's not a simple fix (unlike the last time I mentioned localizability) so I'll let it pass for now.